package.json

```diff
@@ -94,6 +94,11 @@
       "type": "boolean",
       "default": false,
       "description": "When opening a file, always focus an already-existing view of the file even if it's in a another pane."
+    },
+    "alwaysRecursiveCollapse": {
+      "type": "boolean",
+      "default": false,
+      "description": "When collapsing a folder, always collapse the children as well"
```

Comment:
Maybe we should use "descendants" instead of "children" to be more correct? Better yet, "all descendants" to really drive home the recursive nature of this option.

Reply:
agreed. Updated.